addons/editor-theme3/addon.json

```diff
@@ -12,7 +12,13 @@
     {
       "name": "DNin01",
       "link": "https://github.com/DNin01"
-    }
+    },
+    {
+      "name": "Shock59",
+      "link": "https://github.com/shock59"
+    },
+    { "name": "Ashimee", "link": "https://github.com/Ashimee" },
```

Comment:
They deleted their GitHub account, or maybe changed usernames.
Are we sure they want to be credited?